.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         # dart-channel: [stable, dev]
       fail-fast: false
 
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Can you add a comment explaining the use of Mac OS here? Ideally with a link to a tracking bug for the sandbox issue.